### ISSUE TITLE
Stacking different ranks of Protection potions

### ISSUE THREAD

**Issue by Theshame** [elysium-project/server#3039]
I've tested out Greater Fire Protection Potion and Fire Protection Potion. 

Two tests, and both had me absorb more then 3150 fire damage. One test around 3500 dmg, the other around 4000 dmg. 

The buff symbol of the potion I drank first disappears first, no matter if it was greater or lesser.

Are protection pots supposed to stack? I don't want to use it if this is a bug. That would lead to bug abuse. So I want to be sure about this. because if they do stack as they should, this can help very much on bosses like Loatheb, Sapphiron, Kel'Thuzad.

Thank you!

**Comment by connormcgregor** [comment]
They should fix it if its a bug

**Comment by kevmodrome** [comment]
I do not believe protection potions of the same school should stack, there's no evidence of it that I can find at least.
